src/schema/firefox-schemas-import.js

```diff
@@ -198,18 +198,16 @@ inner.mapExtendToRef = (schemas) => {
           allOf: [...extendType.allOf, { $ref: ref }],
         };
       }
-      if (updatedType) {
-        updatedSchemas[namespace] = {
-          ...updatedSchemas[namespace],
-          schema: {
-            ...extendSchema,
-            types: {
-              ...extendSchema.types,
-              [type]: updatedType,
-            },
+      updatedSchemas[namespace] = {
```

Comment:
Looks like `updatedType` is always set so I'm not sure why this if was here in the first place... Thanks for removing it.

Reply:
heh, yeah. That was a good hack to increase coverage :)